website/docs/cloud-docs/integrations/kubernetes/index.mdx

```diff
@@ -1,68 +1,168 @@
+The following example deploys version `1.0.0` of the `hashicorp/module/random` module in the `workspace-name` workspace.
 
+```yaml
+---
+apiVersion: app.terraform.io/v1alpha2
+kind: Module
+metadata:
+  name: my-module
+spec:
+  organization: kubernetes-operator
+  token:
+    secretKeyRef:
+      name: tfc-operator
+      key: token
+  module:
+    source: hashicorp/module/random
+    version: 1.0.0
+  workspace:
+    name: workspace-name
+  variables:
+  - name: string_length
+  outputs:
+  - name: random_string
 ```
-helm upgrade --namespace ${RELEASE_NAMESPACE} ${RELEASE_NAME} hashicorp/terraform
+
+The operator passes the workspace's `string_length` variable to the module and stores the `random_string` outputs as either a Kubernetes secret or a ConfigMap. If the workspace marks the output as "sensitive", the operator stores the `random_string` as a Kubernetes secret; otherwise, the operator stores it as a ConfigMap. The variables must be accessible within the workspace as a workspace variable, workspace variable set, or project variable set.
+
+Refer to the [module API reference](/terraform/cloud-docs/integrations/kubernetes/api-reference#module) for the complete `Module` specification.
+
+### Workspace
+
+Terraform Cloud workspaces organize and manage Terraform configurations. The Terraform Cloud Operator for Kubernetes allows you to create, configure, and manage workspaces directly from Kubernetes.
+
+The following example creates a new workspace named `us-west-development` configured to use Terraform version `1.6.2`. This workspace has two variables, `nodes` and `rds-secret`. The variable `rds-secret` is treated as sensitive, and the operator reads the value for the variable from a Kubernetes secret named `us-west-development-secrets`.
+
+```yaml
+---
+apiVersion: app.terraform.io/v1alpha2
+kind: Workspace
+metadata:
+  name: us-west-development
+spec:
+  organization: kubernetes-operator
+  token:
+    secretKeyRef:
+      name: tfc-operator
+      key: token
+  name: us-west-development
+  description: US West development workspace
+  terraformVersion: 1.6.2
+  applyMethod: auto
+  agentPool:
+    name: ap-us-west-development
+  terraformVariables:
+    - name: nodes
+      value: 2
+    - name: rds-secret
+      sensitive: true
+      valueFrom:
+        secretKeyRef:
+          name: us-west-development-secrets
+          key: rds-secret
+  runTasks:
+    - name: rt-us-west-development
+      stage: pre_plan
 ```
+
+In the above example, the `applyMethod` has the value of `auto`, so Terraform Cloud automatically applies any changes to this workspace. The specification also configures the workspace to use the `ap-us-west-development` agent pool and run the `rt-us-west-development` run task at the `pre_plan` stage.
+
+The operator stores the value of the workspace outputs as Kubernetes secrets or ConfigMaps. If the outputs are marked as "sensitive", they are stored as Kubernetes secrets, otherwise they are stored as ConfigMaps.
```

Comment:
```suggestion
The operator stores the value of the workspace outputs as Kubernetes secrets or ConfigMaps. If the outputs are marked as `sensitive`, they are stored as Kubernetes secrets, otherwise they are stored as ConfigMaps.
```